prime-angular-frontend/src/app/modules/enrolment/pages/access-agreement-history/access-agreement-history.component.ts

```diff
@@ -29,6 +30,10 @@ export class AccessAgreementHistoryComponent implements OnInit {
     this.getAccessTerm();
   }
 
+  public routeTo() {
+    this.router.navigate(['../'], { relativeTo: this.route.parent });
```

Comment:
If you use the BaseEnrolmentPage you can use get routeTo for free through inheritance